### PR TITLE
[INLONG-7067][Manager] Provide MQ cluster info in consumption details

### LINKED ISSUE #7067: [Feature][Manager] Provide MQ cluster info in consumption details
### Description

Data consumption details provide serverUrl, adminUrl and topic information.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

Fixes #7067 

### Motivation

Data consumption details provide serverUrl, adminUrl and topic information.

### Modifications

- Provide the complete topic name, serverUrl and adminUrl when obtaining the topic of data consumption.
- Remove the topic prefix when saving topic information.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java [hunk 1]
```diff
@@ -103,7 +107,16 @@ public InlongConsumeInfo getFromEntity(InlongConsumeEntity entity) {
             ConsumePulsarDTO dto = ConsumePulsarDTO.getFromJson(entity.getExtParams());
             CommonBeanUtils.copyProperties(dto, consumeInfo);
         }
-
+        String groupId = entity.getInlongGroupId();
+        InlongGroupInfo groupInfo = groupService.get(groupId);
+        String clusterTag = groupInfo.getInlongClusterTag();
+        PulsarClusterInfo clusterInfo = (PulsarClusterInfo) clusterService.getOne(clusterTag, null,
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java:113]: Should get all cluster infos under this tag.

**Review by vernedeng — CHANGES_REQUESTED** [review]

**Review by healchow — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java [hunk 2]
```diff
@@ -103,7 +109,22 @@ public InlongConsumeInfo getFromEntity(InlongConsumeEntity entity) {
...
         }
-
+        String groupId = entity.getInlongGroupId();
+        InlongGroupInfo groupInfo = groupService.get(groupId);
+        String clusterTag = groupInfo.getInlongClusterTag();
+        List<ClusterInfo> clusterInfos = clusterService.listByTagAndType(clusterTag, ClusterType.PULSAR);
+        StringBuilder adminUrls = new StringBuilder();
+        StringBuilder serverUrls = new StringBuilder();
+        String topic = entity.getTopic();
+        Preconditions.checkNotEmpty(clusterInfos, "pulsar cluster not exist for groupId=" + groupId);
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java:119]: Put this check after the `List<ClusterInfo> clusterInfos` line.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java [hunk 3]
```diff
@@ -103,7 +109,22 @@ public InlongConsumeInfo getFromEntity(InlongConsumeEntity entity) {
...
+        for (ClusterInfo clusterInfo : clusterInfos) {
+            PulsarClusterInfo pulsarCluster = (PulsarClusterInfo) clusterInfo;
+            adminUrls.append(pulsarCluster.getAdminUrl()).append(InlongConstants.SEMICOLON);
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java:122]: Maybe these clusters are not the same cluster.
For example, cluster A and cluster B do not communicate. If their admin urls and service urls are spliced together, problems will occur.

**Review by healchow — CHANGES_REQUESTED** [review]

**Review by vernedeng — APPROVED** [review]
lgtm

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**